turbo.json

```diff
@@ -7,7 +7,7 @@
             "dependsOn": ["^topo"]
         },
         "build": {
-            "env": ["SUPABASE_ANON_KEY", "SUPABASE_API_KEY"],
+            "env": ["SUPABASE_ANON_KEY", "SUPABASE_API_KEY", "VITE_MIXPANEL_TOKEN", "VITE_ANTHROPIC_API_KEY"],
```

Comment:
Are the supabase keys going to show up correctly without the Vite prefix? I wouldn't think so?